go.mod

```diff
@@ -5,10 +5,10 @@ go 1.16
 require (
 	github.com/containerd/containerd v1.6.3
 	github.com/containernetworking/cni v1.1.0
-	github.com/containers/common v0.47.5-0.20220421111103-112a47964ddb
-	github.com/containers/image/v5 v5.21.1-0.20220421124950-8527e238867c
-	github.com/containers/ocicrypt v1.1.3
-	github.com/containers/storage v1.39.1-0.20220422100603-8996869ae40b
+	github.com/containers/common v0.48.0
+	github.com/containers/image/v5 v5.21.1
```

Comment:
It looks like v5.21.1 is the latest you just built. I'm just confused by the old version, v5.21.1-0.....
I would have thought it would have been v5.21.0-0......

Reply:
Ask @mtrmac

Reply:
AFAICS Podman 4.9 was released with c/image 5.19, so we don’t _need_ a new 5.Y version; we had a 5.21.0 release but not a Podman-related branch, so we could now release a 5.21.1, and branch off that.

If that’s mistaken, I’m fine with tagging a new 5.22.0 on top.